@sairavuri-sudo The change is almost ready to merge, could you fix the following lint error from the CI job?
```
ruff.....................................................................Failed
- hook id: ruff
- exit code: 1

Error: tests/genai/judges/adapters/test_gateway_adapter.py:1147:23: error[PT011] `pytest.raises(mlflow.exceptions.MlflowException)` is too broad, set the `match` parameter or use a more specific exception
```

--- a/tests/genai/judges/adapters/test_gateway_adapter.py
+++ b/tests/genai/judges/adapters/test_gateway_adapter.py
@@ -1144,7 +1144,7 @@ def test_invoke_and_handle_tools_forwards_workspace_header():
             "mlflow.genai.judges.adapters.gateway_adapter.send_chat_request",
             side_effect=capture_and_raise,
         ),
-        pytest.raises(MlflowException),
+        pytest.raises(MlflowException, match="stop"),
     ):
         adapter._invoke_and_handle_tools(
             provider="gateway",

